src/lib/detection_algorithms/cntk/test/test_exporter.js

```diff
@@ -0,0 +1,19 @@
+const YoloExporter = require('../main.js').Exporter;
+const path = require('path');
+
+var exporter = new YoloExporter(path.join(__dirname, 'export_test'), ['cat', 'dog', 'mouse'], 1000, 1000);
```

Comment:
Are you using it to test? It seems like it's just a left over from the copy&paste..probably should be removed